book/website/collaboration/leadership/leadership-building.md

```diff
@@ -0,0 +1,103 @@
+Human relationships are complicated, and leaders must make consensus-led decisions, however, sometimes those decisions may not align with the scope and goals of the project. 
+This is where, providing a clear governance process can help leaders in avoiding arbitrary decision-making, bring alignments and enable transparent governance.
+- **Knowledge and skill building**: Leaders have deep knowledge of their projects, and teams and continue to build skills for themselves and others as needed.
+They bring systems-level thinking and familiarity with the whole organisation where they lead, and the system within which they work with. 
+Leaders’ goals also include identifying opportunities for and directly building complementary skills for the entire team, encouraging problem-solving as a team, developing a workflow that enhances the efficiency of the team (for example reusable and reproducible processes) and sharing knowledge with each other.
+At the core of this is building diverse teams, bringing perspectives from diverse cultural backgrounds and working with them to solve problems faster.
+- **Consensus**: Leaders act on as much consensus as possible, even when they do not agree with the decision individually.
+A healthy way to navigate the difference is to create spaces to have informed conversations where multiple perspectives are explored and the team can assess the pro and cons of each possible decision. 
+This also means that leaders provide a process for how decisions are made, and invite inputs from their team members on improving it further.
+Sometimes it may mean that they reach out to the people who don't actively engage, or feel left out of the consensus to make decisions that are best for the maximum number of people affected by it.
+- **Trust**: Healthy leadership trust in the skills and ability of people around them and identify the best fit for everyone to succeed.
+They are aware of their own biases and avoid them intentionally, often by providing space to listen to different thoughts and perspectives. 
+They guide people to adapt to changes and transformations particularly when leading large projects, with many moving parts and teams.
+- **Delegation**: Leaders are to lead by example, but they will be unable to do everything and should not do everything either. 
+When a leader is in charge of too many aspects of a project they will become a bottleneck and blocker to the emergence of other open leaders.
+Delegation goes hand in hand with building a trustworthy environment, this means that they communicate clearly the overarching goals, create smaller milestones and delegate tasks meaningfully so different people can lead different parts of the work, nurturing healthy leadership habits within their teams.
+- **Power dynamics**: Power dynamics are relationships between people in an organisation with varying levels of privileges. 
+Power dynamics are often created by leaders.
```

Comment:
That's a good point.
I guess what I wanted to say is that Leader are the source of power dynamics because of traditional heirarchy they exist in. We can simply use what you said there.

Reply:
```suggestion
Leaders hold power because of their position in the project (formally or socially) and hence have significant influence over how those power dynamics are experienced or manipulated in their teams and communities.
```